README.md

```diff
@@ -30,8 +30,11 @@ We will create a Boxen for this project, but for now to get started:
 4. Run `overcommit --install` to install git hooks into the repo
 5. Run `bower install`
 6. Run `grunt`. Ensure that the *latest* pattern library is in the same folder as this repo.
-Running grunt will migrate the css over from the patter library.
+Running grunt will migrate the css over from the pattern library. You can ignore the warning that says `Unable to match 1 pattern.`.
 6. Run `rake bower:install`
```

Comment:
Looks like there are two 6. bullet points. Given that it's markdown, I think you can convert them all the 1. and the numbering should work.